packages/neon/neon/lib/src/utils/bloc.dart

```diff
@@ -1,7 +1,7 @@
 part of '../../neon.dart';
 
 abstract class Bloc {
-  void dispose();
+  void dispose() {}
```

Comment:
what's the reason?

Ideally I'd like to mark both `Bloc` as an interface with `abstract interface class ...` unless it isn't really an interface :thinking:

Reply:
I don't think every bloc needs to be disposable. This no longer forces inheritors to implement the function. Adding the interface modifier seems like a good idea though